WordPress/Classes/ViewRelated/Gutenberg/GutenbergWeb/GutenbergWebViewController.swift

```diff
@@ -82,6 +82,12 @@ class GutenbergWebViewController: GutenbergWebSingleBlockViewController, WebKitA
         ].compactMap { $0 }
     }
 
+    override func onGutenbergReadyStyles() -> [WKUserScript] {
```

Comment:
Just wanted to flag [this comment from the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/37221#discussion_r771089838) about the function name.